### ISSUE TITLE
Workflow file information in Service creation

### ISSUE THREAD

**Issue by SekarRaj** [agorago/wego-gen#1]
```sh
➜  agorago wego-gen/bin/gen.sh sample-api 
1) Generate a state service
2) Generate a deployable
3) Generate API & Service
Please enter your choice: (or quit to exit the program) 1
Workflow File:
Workflow File:
Workflow File:
```

What is the expected value required for the workflow file name.

**Comment by rajakolluru** [comment]
The workflow file is a json that specifies the workflow. For example consider the workflow file in https://github.com/agorago/wego/blob/master/stm/test.json
This, for example represents a workflow that takes an order from an initial state of "created" to "confirmed" to "fulfilled" to "closed". The wego-gen generates all the workflow actions for each of these transitions to happen. I will give you an example and commit it to this repo

**Comment by SekarRaj** [comment]
> The workflow file is a json that specifies the workflow. For example consider the workflow file in https://github.com/agorago/wego/blob/master/stm/test.json
> This, for example represents a workflow that takes an order from an initial state of "created" to "confirmed" to "fulfilled" to "closed". The wego-gen generates all the workflow actions for each of these transitions to happen. I will give you an example and commit it to this repo

So the workflow must be first defined before creating the service itself. Is that correct?

**Comment by rajakolluru** [comment]
That is true Raj. This is because once you document the workflow we can then generate the stubs for all the workflow actions and register these actions automatically into WeGO workflow
